src/util/ConfigParser.js

```diff
@@ -2,9 +2,9 @@ const cosmiconfig = require('cosmiconfig');
 const ArgumentParser = require('argparse').ArgumentParser;
 
 class ConfigParser {
-    parseConfig() {
+    parseConfig(logger) {
         let args = this.parseArgs();
-        return { paths: this.parsePaths(args.config), args };
+        return { paths: this.parsePaths(args.config, logger), args };
```

Comment:
Would it be better to set the logger as an attribute of the class instead of passing it as param?

Up to debate tbh